src/Symfony/Component/Console/Application.php

```diff
@@ -618,8 +618,19 @@ public function find(string $name)
             $commands = preg_grep('{^'.$expr.'}i', $allCommands);
         }
 
+        $nameSwapped = implode(':', array_reverse(explode(':', $name)));
```

Comment:
What about commands with 3 parts? like `doctrine:schema:update`

Reply:
This change only applies to build in core commands for now.

Reply:
The way it's implemented here does not take only core commands
And even in the core, there are commands with 3 parts, like `cache:pool:*`

Reply:
Let me check again and see what I can do about that.